packages/website/src/templates/doc-page.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+import { graphql } from 'gatsby'
+import { MDXProvider } from '@mdx-js/react'
+import { MDXRenderer } from 'gatsby-plugin-mdx'
+import * as FC from '@containous/faency'
+import Layout from '../components/Layout'
+import CodeBlock from '../components/CodeBlock'
+import { SystemPropsTable } from '../components/SystemPropsTable'
+import { PropsTable } from '../components/PropsTable'
+import { ThemeSection } from '../components/ThemeSection'
+
+export const components = {
+  ...FC,
+  pre: props => <div {...props} />,
+  code: CodeBlock,
+  table: FC.Table,
+  thead: FC.Thead,
+  tbody: FC.Tbody,
+  tfoot: FC.Tfoot,
+  tr: FC.Tr,
+  th: FC.Th,
+  td: FC.Td,
+  inlineCode: props => <FC.Chip variant="blue" {...props} />,
+  SystemProps: SystemPropsTable,
+  PropsTable: PropsTable,
+  h1: props => <FC.Heading {...props} size={3} mt={8} mb={30} />,
+  h2: props => <FC.Heading {...props} as="h2" size={2} mt={8} mb={26} />,
+  h3: props => <FC.Heading {...props} as="h3" size={1} mt={7} mb={22} lineHeight={1} />,
+  h4: props => <FC.Heading {...props} as="h4" fontWeight={500} size={0} mt={6} mb={1} lineHeight={2} />,
+  p: props => <FC.Text {...props} as="p" size={3} lineHeight={3} textColor={FC.theme.colors.grays[7]} />,
+  Prop: props => (
+    <FC.Box mb={3}>
+      <FC.Text as="p" size={2} lineHeight={2} textColor={FC.theme.colors.grays[7]}>
+        {props.isOptional && 'optional'} <code>{props.children}</code>
+      </FC.Text>
+      {props.default && (
+        <FC.Text as="p" size={2} lineHeight={2} textColor="gray700">
+          default <code>{props.default}</code>
+        </FC.Text>
+      )}
+    </FC.Box>
+  ),
+  ThemeSection,
+}
+
+console.log('components', components)
+
+function DocPageTemplate({ data, location, ...props }) {
+  const children = <MDXRenderer children={data.mdx.body} />
+
+  return (
+    <Layout pathname={location.pathname}>
+      <Helmet>
+        <title>Faency — {data.mdx.frontmatter.title}</title>
+        <meta property="og:title" content={`Faency — ${data.mdx.frontmatter.title}`} />
+        <meta property="og:description" content={data.mdx.frontmatter.description} />
+        <meta name="twitter:title" content={`Faency — ${data.mdx.frontmatter.title}`} />
+        <meta name="twitter:description" content={data.mdx.frontmatter.description} />
+      </Helmet>
+
+      <MDXProvider components={components}>
+        <FC.Box>
+          <FC.Heading size={4} mb={20} lineHeight={4}>
+            {data.mdx.frontmatter.title}
+          </FC.Heading>
+          <FC.Heading size={1} mb={7} as="h2" lineHeight={3} textColor={FC.theme.colors.grays[5]}>
+            {data.mdx.frontmatter.description}
+          </FC.Heading>
+          {children}
+        </FC.Box>
+      </MDXProvider>
+    </Layout>
+  )
+}
+
+export default DocPageTemplate
+
+// eslint-disable-next-line no-undef
```

Comment:
I'm probably missing something, but what is undefined that needs to be ignored in the next line?